crowdkit/aggregation/multilabel/binary_relevance.py

```diff
@@ -13,23 +13,35 @@
 
 @attr.s
 class BinaryRelevance(BaseClassificationAggregator):
-    r"""Simple aggregation algorithm for multi-label classification.
-
-    Binary Relevance is a straightforward approach for multi-label classification aggregation:
-    each label is treated as a class in binary classification problem and aggregated separately using
-    aggregation algorithms for classification, e.g. Majority Vote or Dawid Skene.
+    r"""The **Binary Relevance** algorithm is a simple aggregation algorithm for the multi-label classification.
+
+    Binary Relevance is a straightforward approach for the multi-label classification aggregation:
+    each label is represented as a class in the binary classification problem and aggregated separately using
+    aggregation algorithms for classification (e.g., Majority Vote or Dawid-Skene). Specifically,
+    for each class label $λ_j$, Binary Relevance derives a binary training set $D_j$ from the original
+    multi-label training set $D$ in the following way:
+    $$
+    D_j = {(x^i, y_j^i) | 1 <= i <= m}.
```

Comment:
Please check i and j meanings; now I guess "1 <= j <= m" is correct since we have j class labels (this follows from "for each class label $λ_j$")

Reply:
Please add the meaning of the m parameter and a brief explanation of `y_j^i`

Reply:
Please add the meaning of the m parameter and a brief explanation of `y_j^i`